integrations/jenkins/client.py

```diff
@@ -119,3 +119,29 @@ async def get_single_resource(self, resource_url: str) -> dict[str, Any]:
         )
         response.raise_for_status()
         return response.json()
+
+    async def get_users(self) -> AsyncGenerator[list[dict[str, Any]], None]:
+        page_size = 5
```

Comment:
Any reason why the page size is very small?

Reply:
Testing. I should make that 50.